### PR TITLE
chore: upgrade to polkadot 0.9.19

### PR DESCRIPTION
**Pull request by sander2**
- Changed edition from 2018 to 2021 (prompted because pallets gave errors due to a change in substrate. We could have fixed it by doing some imports, but it's nice to upgrade anyway)
- Updated for the change in payment pallet: https://github.com/paritytech/substrate/pull/10785
- Some CLI arguments (particularly benchmarking) required changes
- Kusama xcm execution fees increased

### CODE REVIEW

File: parachain/runtime/runtime-tests/src/relaychain/kusama_cross_chain_transfer.rs [hunk 1]
```diff
@@ -57,7 +57,7 @@ fn transfer_to_relay_chain() {
     });
 
     KusamaNet::execute_with(|| {
-        let xcm_fee = 106_666_660;
+        let xcm_fee = 165_940_672;
```
Comment (gregdhill) [inline parachain/runtime/runtime-tests/src/relaychain/kusama_cross_chain_transfer.rs:60]: Where did this magic number come from? Will this test break on upgrade and if so how do we change this?

File: parachain/src/command.rs [hunk 2]
```diff
@@ -286,26 +287,105 @@ pub fn run() -> Result<()> {
...
+                            cmd.run(config, partials.client.clone(), db, storage)
+                        })
+                    } else {
+                        Err("Chain doesn't support benchmarking".into())
+                    }
+                }
+                BenchmarkCmd::Overhead(_) => Err("Unsupported benchmarking command".into()),
             }
         }
+        // Some(Subcommand::Benchmark(cmd)) => {
```
Comment (gregdhill) [inline parachain/src/command.rs:371]: There will be conflicts now due to the `instant-seal` changes, also did you mean to uncomment or delete these lines?

File: parachain/runtime/runtime-tests/src/relaychain/kusama_cross_chain_transfer.rs (hunk 1 above)
Comment (sander2) [reply parachain/runtime/runtime-tests/src/relaychain/kusama_cross_chain_transfer.rs:60]: The fee calculations changed on kusama: https://github.com/paritytech/polkadot/pull/5202, in particular the `ExtrinsicBaseWeight`. See https://github.com/interlay/interbtc-ui/pull/288 for more detail

Comment (gregdhill) [reply parachain/runtime/runtime-tests/src/relaychain/kusama_cross_chain_transfer.rs:60]: @sander2 can you add that as a comment? Will make it easier to upgrade in the future.

**Pull request merged**